src/main/resources/db/1.0_create_table_causal_data.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.7"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.7
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.7.xsd">
+    <changeSet id="v1-create-table-causal-data" author="dschuler36">
+        <comment>Creates the initial version of the Causal table</comment>
+        <createTable tableName="CAUSAL_DATA"
+                     remarks="Signifies whether a given product was featured in the weekly mailer or was part of an in-store display.">
+            <column autoIncrement="true"
+                    name="ID"
+                    type="BIGINT"
+                    remarks="Number that uniquely identifies each product">

```

Comment:
I think this is a surrogate key, so if you could just update the remarks to reflect that.